packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -98,17 +85,14 @@ class FirebaseFirestoreWeb extends FirebaseFirestorePlatform {
   @override
   Future<T?> runTransaction<T>(TransactionHandler<T> transactionHandler,
       {Duration timeout = const Duration(seconds: 30)}) async {
-    try {
-      await _webFirestore.runTransaction((transaction) async {
-        return transactionHandler(
-            TransactionWeb(this, _webFirestore, transaction!));
-      }).timeout(timeout);
-      // Workaround for 'Runtime type information not available for type_variable_local'
-      // See: https://github.com/dart-lang/sdk/issues/29722
-      return null;
-    } catch (e) {
-      throw getFirebaseException(e);
-    }
+    await guard(() => _webFirestore.runTransaction((transaction) async {
```

Comment:
This function looks broken. It appears that it now _always_ return `null`

Reply:
Oh my bad. But then I don't get why this function returns `T`, if it justs never return anything